magic/oracle.py

```diff
@@ -36,17 +36,17 @@ def load_cards(names: Iterable[str] = None, where: Optional[str] = None) -> List
     else:
         setnames = set()
     if setnames:
-        names_clause = 'c.name IN ({names})'.format(names=', '.join(sqlescape(name) for name in setnames))
+        names_clause = 'name IN ({names})'.format(names=', '.join(sqlescape(name) for name in setnames))
     else:
         names_clause = '(1 = 1)'
     if where is None:
         where = '(1 = 1)'
     sql = multiverse.cached_base_query('({where} AND {names})'.format(where=where, names=names_clause))
-    rs = db().select(sql)
-    if setnames and len(setnames) != len(rs):
-        missing = setnames.symmetric_difference([r['name'] for r in rs])
-        raise TooFewItemsException('Expected `{namelen}` and got `{rslen}` with `{names}`.  missing=`{missing}`'.format(namelen=len(setnames), rslen=len(rs), names=setnames, missing=missing))
-    return [Card(r) for r in rs]
+    rows = db().select(sql)
```

Comment:
The renaming of the variables here feels a little pointless. `rs` is a pretty traditional name for a resultset/recordset. In general, also, we use 'x' and 'xs' for local vars throughout the codebase.